test/resources/case32_secret_stringdata/case32_create_secret.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: ConfigurationPolicy
+metadata:
+  name: case32config
+spec:
+  object-templates:
+  - complianceType: musthave
+    objectDefinition:
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: htpasswd-secret
+        namespace: managed
+      stringData:
+        htpasswd: |
+          alice:$QGJM$2qku.rrSEWRFD/tjOf.dZQ1q.RaR5tTmfNyZn0$1uI6eDx/Ont37ws
+          bob:S1KPovd0N/iDb9$dlTBH7weXG5rQV$5EPf9KrbJjw2HXy35ujhPw2y0$SIto
+          cadence:W$MeWCMWuKi0ApUyZ8byG5$DNL.3FD$c72kFhyZIX8Sf0yJH2AEOGFsRgEHJ
+  pruneObjectBehavior: None
+  remediationAction: enforce
+  severity: medium
+  
```

Comment:
There's some non-newline whitespace on this line
```suggestion

```

Reply:
goood catch!!

Reply:
Changed